src/Upgrader/Tasks/UpgraderTask.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Tasks;
```

Comment:
@alexanderM91  Why not Upgrader\Infrastructure\Tasks ? or we will use "Upgrader" as a fully infrastructure level? without Application and Domain

Reply:
Not blocker